client-gateway/bot/drive_ops_bot.py

```
@@ -0,0 +1,151 @@
+        parse_mode='Markdown',
+        reply_markup=types.ReplyKeyboardRemove() # Ховаємо головне меню
+    )
+    bot.register_next_step_handler(msg, process_pickup_step)
+
+def process_pickup_step(message):
+    chat_id = message.chat.id
+    address = message.text
+
+    # Валідація Mandatory field
+    if not is_valid_address(address):
+        msg = bot.send_message(chat_id, "❌ Адреса занадто коротка або некоректна. Спробуйте ще раз:")
```

Ruff (0.14.10)

String contains ambiguous `а` (CYRILLIC SMALL LETTER A). Did you mean `a` (LATIN SMALL LETTER A)?

(RUF001)